llama_stack/distribution/library_client.py

```diff
@@ -254,8 +254,12 @@ async def request(
         stream=False,
         stream_cls=None,
     ):
-        if not self.endpoint_impls:
-            raise ValueError("Client not initialized")
+        try:
```

Comment:
I think a better fix is to just initialize `endpoint_impls = None` in the constructor so that the correct ValueError is raised here instead of a weird `AttributeError`. Trapping an AttributeError is kind of unpredictable anyhow.